src/resolvers/farcaster.ts

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+import { getAddress, isAddress } from '@ethersproject/address';
+import { resize } from '../utils';
+import { max } from '../constants.json';
+import { fetchHttpImage, axiosDefaultParams } from './utils';
```

Comment:
Can you remove this, this is not used, also please run the lint script
```suggestion
import { fetchHttpImage } from './utils';
```

Reply:
@bonustrack can't run the lint script as yarn fails with issues in regards to module 'canvas' node-gyp errors

Reply:
ran eslint manually to get linting